sources/Inputs/src/Configuration.cpp

```diff
@@ -41,6 +42,28 @@ updateValue(T& value, const boost::property_tree::ptree& tree, const std::string
     value = value_opt->get_value<T>();
   }
 }
+
+/**
+ * @brief Helper function to update the active power compensation parameter, if the value of ActivePowerCompensation
+ * is not one of the three available, then the default value is used 
```

Comment:
Please specify that activePowerCompensation is not set when the parameter is not found (the default value used only because it as set to this value before this method).

Reply:
Should I also put in the comment that the default value is used because it was set beforehand ?